horton/gbasis/ints.h

```diff
@@ -63,8 +63,32 @@ class GB2KineticIntegral: public GB2Integral {
 /** @brief
  Compute the nuclear attraction integrals in a Gaussian orbital basis.
  */
-class GB2NuclearAttractionIntegral: public GB2Integral {
+class GB2AttractionIntegral: public GB2Integral {
     private:
+    /** @brief
```

Comment:
This piece of documentation is not correctly indented. Also, every variable should be documented separately. There are a few ways to do this:

- Short:

      double* charges;  //!< Array with values of the nuclear charges.

- Intermediate:

      //! Array with values of the nuclear charges.
      double* charges;

- (too) Long

      /** @brief
            Array with values of the nuclear charges.
      */
      double* charges;